utils/build_system/create_test_release_jobs.py

```diff
@@ -75,6 +75,8 @@ def create_directory(self, name: Path | str, display_name: str):
             new_path = str(Path(self.base_job_dir) / name)
 
             if self.jenkins.job_exists(new_path):
+                if name in ['scylla-master', 'scylla-enterprise']:
```

Comment:
can you add a comment why we do it for future reference?

Reply:
We can probably remove, it's just slow the process a bit, the update of such folder with so many jobs under it, seems to be taking a while.

And I don't want to override anything in a folder that wasn't created with SCT.

Reply:
added a comment saying exactly this